deploy/markets.ts

```diff
@@ -102,15 +103,35 @@ const initializeMarkets: DeployFunction = async (hre) => {
       })
     }
 
-    const tTokenStrategy = await deploy({
-      hre,
-      contract: market.strategy.name,
-      indent: 4,
-    })
+    let tTokenStrategy: Contract
+
+    if (market.strategy.name == 'TTokenCompoundStrategy_2') {
+      tTokenStrategy = await deploy({
+        hre,
+        contract: market.strategy.name,
+        indent: 4,
+        args: [await (await getNamedSigner('deployer')).getAddress()],
```

Comment:
we should add a named account to the `hardhat.config.ts`

Reply:
added as `gnosisSafe`